src/Valit/Rules/ValitRuleExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Linq;
+using Valit.Exceptions;
+using Valit.Result;
+
+namespace Valit.Rules
+{
+    internal static  class ValitRuleExtensions
+    {
+        internal static IValitResult ValidateRules<TObject>(this IEnumerable<IValitRule<TObject>> rules, IValitStrategy strategy, TObject @object) where TObject : class
+        {
+            rules.ThrowIfNull();
+
+            var result = ValitResult.Success;
```

Comment:
This seems like an `enum`  to me, maybe it would be clearer to do `new ValitResult()`?

Reply:
You can always hoover on var so it's clear that's ``IValitStrategy`` object :D

Reply:
True 😄